### PR TITLE
Fix time reporting on Windows

### PR DESCRIPTION
**Pull request by compnerd**

### CODE REVIEW

**Comment by compnerd** [comment]
CC: @millenomi @jakepetroules @phausler @parkera

**Comment by compnerd** [comment]
@swift-ci please test

File: CoreFoundation/NumberDate.subproj/CFDate.c [hunk 1]
```diff
@@ -40,6 +40,7 @@
 
 const CFTimeInterval kCFAbsoluteTimeIntervalSince1970 = 978307200.0L;
 const CFTimeInterval kCFAbsoluteTimeIntervalSince1904 = 3061152000.0L;
+const CFTimeInterval kCFAbsoluteTimeIntervalSince1601 = 12622780800.0L;
```
Comment (millenomi) [inline CoreFoundation/NumberDate.subproj/CFDate.c]: This needs to be `static`.

Comment (compnerd) [reply CoreFoundation/NumberDate.subproj/CFDate.c]: Actually, shouldn't all three of those be `static`?

Comment (millenomi) [reply CoreFoundation/NumberDate.subproj/CFDate.c]: The other two are API.

Comment (millenomi) [reply CoreFoundation/NumberDate.subproj/CFDate.c]: https://developer.apple.com/documentation/corefoundation/kcfabsolutetimeintervalsince1970
https://developer.apple.com/documentation/corefoundation/kcfabsolutetimeintervalsince1904

**Comment by compnerd** [comment]
@swift-ci please test and merge

**Comment by compnerd** [comment]
@swift-ci please test and merge

**Pull request merged**